src/app/shared/hero/hero.component.css

```diff
@@ -36,7 +36,7 @@
 }
 .toolbar section {
   display: flex;
-  align-items: center;
```

Comment:
I adjusted the toolbar layout to align to the bottom. The last saved and published at info not being aligned bothered me. I also opened bug #199 for 2 line titles.